src/arrayMethodSort.js

```diff
@@ -4,8 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (String(a) < String(b)) {
+        return -1;
+      }
+
+      if (String(a) > String(b)) {
+        return 1;
+      }
+
+      return 0;
+    },
+  ) {
+    let swapped;
+
+    for (let i = 0; i < this.length; i++) {
+      swapped = false;
+
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+          swapped = true;
+        }
```

Comment:
Array destructuring is used here for swapping elements, which is a clean and modern approach. Just be aware that this can be less performant than using a temporary variable in some JavaScript engines due to the creation of temporary arrays.